doc/man3/EVP_PKEY_new.pod

```diff
@@ -219,7 +219,19 @@ general private key without reference to any particular algorithm.
 The structure returned by EVP_PKEY_new() is empty. To add a private or public
 key to this empty structure use the appropriate functions described in
 L<EVP_PKEY_set1_RSA(3)>, L<EVP_PKEY_set1_DSA(3)>, L<EVP_PKEY_set1_DH(3)> or
-L<EVP_PKEY_set1_EC_KEY(3)>.
+L<EVP_PKEY_set1_EC_KEY(3)> for legacy key types implemented in internal
+OpenSSL providers.
+
+For fully provider-managed key types (see L<provider-keymgmt(7)>),
+possibly implemented in external providers, use functions such as
+L<EVP_PKEY_set1_encoded_public_key(3)> or L<EVP_PKEY_set_params(3)>
```

Comment:
But EVP_PKEY_set_params() does not work to import private key data. You really need to use EVP_PKEY_fromdata().

Reply:
For some reason I was of the opinion that that is possible. But it's obviously at the discretion of a provider and clearly not a [Common Information Parameter](https://docs.openssl.org/3.5/man7/provider-keymgmt/#common-information-parameters). Fixed & Thanks for the correction.